### DISCUSSION TITLE
Can this be used to create proposals dynamically?

### DISCUSSION THREAD

**Discussion by boringcommerce** [documenso/documenso#1124] [General]
Hello,

I'm a little confused. I wanted to replace pandadoc in order to auto-create proposals using API.

Is this a good option for the same? Or creating one on google doc, and then transfering it to documenso a better option? Kindly advice.

**Comment by ElTimuro** [comment]
@boringcommerce 

- You can upload a doc, but we don't have a complete proposal editor yet
- If you wan to send advanced proposals I would recommend creating them in Google Docs and uploading the complete proposal and just add a signature field (and maybe a date field)

**Reply by boringcommerce** [reply]
That sounds fine. But will this be able to automatically take the file from google doc to documenso (using API), and via using make.com/ zapier or some workflow automation tool?

**Reply by ElTimuro** [reply]
- We support Zapier; since they have google docs, it should work
- Currently we only support PDF, so generate a pdf should be part of the flow
- We don't support make.com yet

**Reply by boringcommerce** [reply]
No like is there a direct API where I can send the document to?

**Reply by ElTimuro** [reply]
We have both, not sure what you mean by direct:
- We have an API: https://app.documenso.com/api/v1/openapi
- And a Zapier Integration: https://zapier.com/apps/documenso/integrations
